diff --git a/scripts/babbage/mkfiles.sh b/scripts/babbage/mkfiles.sh
index 6ed8d78fb..117e5f020 100755
--- a/scripts/babbage/mkfiles.sh
+++ b/scripts/babbage/mkfiles.sh
@@ -32,12 +32,12 @@ case $UNAME in
 esac
 
 CARDANO_CLI="${CARDANO_CLI:-cardano-cli}"
-NETWORK_MAGIC=42
+NETWORK_MAGIC=1
 SECURITY_PARAM=10
 NUM_SPO_NODES=3
 INIT_SUPPLY=12000000
 START_TIME="$(${DATE} -d "now + 30 seconds" +%s)"
-ROOT=example
+ROOT=/cluster
 mkdir -p "${ROOT}"
 
 cat > "${ROOT}/byron.genesis.spec.json" <<EOF
@@ -135,7 +135,7 @@ mv "${ROOT}/genesis.alonzo.json" "${ROOT}/genesis/shelley/genesis.alonzo.json"
 mv "${ROOT}/genesis.conway.json" "${ROOT}/genesis/shelley/genesis.conway.json"
 mv "${ROOT}/genesis.json" "${ROOT}/genesis/shelley/genesis.json"
 
-jq --raw-output '.protocolConsts.protocolMagic = 42' "${ROOT}/genesis/byron/genesis-wrong.json" > "${ROOT}/genesis/byron/genesis.json"
+jq --raw-output ".protocolConsts.protocolMagic = ${NETWORK_MAGIC}" "${ROOT}/genesis/byron/genesis-wrong.json" > "${ROOT}/genesis/byron/genesis.json"
 
 rm "${ROOT}/genesis/byron/genesis-wrong.json"
 
@@ -241,7 +241,7 @@ CARDANO_NODE="\${CARDANO_NODE:-cardano-node}"
   --config                          '${ROOT}/configuration.yaml' \\
   --topology                        '${ROOT}/${NODE}/topology.json' \\
   --database-path                   '${ROOT}/${NODE}/db' \\
-  --socket-path                     '$(sprocket "${ROOT}/${NODE}/node.sock")' \\
+  --socket-path                     '$(sprocket "/node-ipc/${NODE}/node.socket")' \\
   --shelley-kes-key                 '${ROOT}/${NODE}/kes.skey' \\
   --shelley-vrf-key                 '${ROOT}/${NODE}/vrf.skey' \\
   --byron-delegation-certificate    '${ROOT}/${NODE}/byron-delegation.cert' \\
